src/bindings/python/src/compatibility/openvino/inference_engine/ie_api_impl.cpp

```diff
@@ -31,12 +31,11 @@ std::map<std::string, InferenceEngine::Layout> layout_map = {{"ANY", InferenceEn
                                                              {"CN", InferenceEngine::Layout::CN},
                                                              {"NCDHW", InferenceEngine::Layout::NCDHW},
                                                              {"BLOCKED", InferenceEngine::Layout::BLOCKED}};
-#define stringify(name) #name
 #define IE_CHECK_CALL(expr)                           \
     {                                                 \
         auto ret = (expr);                            \
         if (ret != InferenceEngine::StatusCode::OK) { \
-            IE_THROW() << response.msg;               \
+            IE_THROW_G(response.msg);                 \
```

Comment:
I didn't expect that we will change the semantic of IE_THROW function...
It looks like we have a lot of changes in the code which will be removed soon. Can we avoid it? Right now I see a big effort just to support something in old API. I would like to avoid it.

Reply:
This PR #18284 has been merged. I think this PR can be closed as it impact project to in too many places.